installbuilder/datafiles/Base_DSC.data

```diff
@@ -363,7 +374,7 @@ if [ "$1" = "0" -o "$1" = "remove" -o "$1" = "purge" ]; then
     done
 fi
 
-%Postuninstall_999
+%Postuninstall_899
```

Comment:
>Postuninstall_899 [](start = 1, length = 17)

what is the number for?

Reply:
Just the order of operations.

---
In reply to: [263558913](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/541#discussion_r263558913) [](ancestors = 263558913)